util/misc.go

```diff
@@ -554,7 +554,18 @@ func LoadTLSCertificates(ca, key, cert string, autoTLS bool, rsaKeySize int) (tl
 // IsTLSExpiredError checks error is caused by TLS expired.
 func IsTLSExpiredError(err error) bool {
 	err = errors.Cause(err)
-	if inval, ok := err.(x509.CertificateInvalidError); !ok || inval.Reason != x509.Expired {
+	switch inval := err.(type) {
+	case x509.CertificateInvalidError:
+		if inval.Reason != x509.Expired {
+			return false
+		}
+	case *tls.CertificateVerificationError:
```

Comment:
It is relation with ```go1.20```.

```
For a handshake failure due to a certificate verification failure, the TLS client and server now return an error of the new type [CertificateVerificationError](https://tip.golang.org/pkg/crypto/tls/#CertificateVerificationError), which includes the presented certificates.
```

Reply:
Although we use bazel to build/test code with go1.20, we have some tasks with go1.19. I have to skip this test after all upgrading the go1.20